unison-src/tests/methodical/dots.u

```diff
@@ -14,15 +14,15 @@ id2 = id . id
 -- foo = id .zonk
 
 -- You can define qualified functions
-(base.function..) f g x = f (g x)
+(base.function.`.`) f g x = f (g x)
```

Comment:
Even with the relaxed rules proposed in the previous comment, this would still need the ``` `` ```s, because it's got other `.` in it, which would mean it's a path-qualified name.